support/index.html

```diff
@@ -6,7 +6,7 @@
 <div class="section" id="report-a-bug">
 <h1>Report a bug</h1>
 <p>If you encounter a bug in IronPython, or would like to ask for a feature,
-please <a class="reference external" href="https://github.com/IronLanguages/main/issues/new">submit an issue</a> on the
+please <a class="reference external" href="https://github.com/IronLanguages/">submit an issue</a> on the
 <a class="reference external" href="https://github.com/IronLanguages/main">IronPython GitHub project</a>. Take a
```

Comment:
Probably need this one updated too, can you update it as well?

Reply:
@slide I changed it to point to ironpython2 instead.